Update 5.46

blmaier opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

blmaier committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.03.09.04.35

blmaier committed:
fix license

https://spdx.org/licenses/BSD-2-Clause-Darwin.html

blmaier committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.03.09.04.35

blmaier committed:
update upstream url to http

blmaier committed:
update to 5.46

blmaier committed:
add stdlib('c')

blmaier committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.03.09.04.35

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12263897839](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12263897839). Examine the logs at this URL for more detail.</sub>

blmaier commented on recipe/meta.yaml:
```diff
@@ -77,7 +85,7 @@ outputs:
   home: http://darwinsys.com/file
   summary: Fine Free File Command
-  license: BSD-2-Clause
+  license: BSD-2-Clause-Darwin
```
Note this is the correct SPDX license identifier. It's a identifier specifically for `file`.
https://spdx.org/licenses/BSD-2-Clause-Darwin.html

The linter warns about it as it appears to be more recently added to SPDX. I opened a PR to smithy to update the SPDX license list.
https://github.com/conda-forge/conda-smithy/pull/2177

blmaier committed:
rename top-level to fix file tests

The tests under the `file` subpackage are not running.

The conda forge docs recommend the top-level package's name should be
different from the subpackage's names. Changing the top-level fixes the
tests so they start running.

https://conda-forge.org/docs/maintainer/knowledge_base/#common-pitfalls-with-outputs

blmaier committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.03.09.04.35

blmaier commented:
@chrisburr @mariusvniekerk @mrakitin could I get a review on this?

blmaier commented:
Hi @conda-forge/core, this repo appears to be abandoned, could you look at it?

h-vetinari commented on recipe/meta.yaml:
```diff
@@ -77,11 +80,12 @@ outputs:
 
 extra:
+  feedstock-name: {{ name|lower }}
   recipe-maintainers:
```
Please add yourself to maintainers and rerender

h-vetinari reviewed (commented):
Thanks for reviving this feedstock!

blmaier committed:
add blmaier as maintainer

blmaier committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.44.9, and conda-forge-pinning 2024.12.10.18.05.37
